### ISSUE TITLE
Full planet setup, path blocked by barrier

### ISSUE THREAD

**Issue by monikgrover** [graphhopper/graphhopper#1598]
Hi, I have done a full planet setup but having issues with routes, not giving exact results as in https://graphhopper.com/maps. 
for eg:-

http://35.246.8.72:8989/maps/?point=50.549126%2C-4.144668&point=Two%20Bridges%20Hotel%2C%20PL20%206SW%2C%20West%20Devon%2C%20United%20Kingdom&locale=en-US&vehicle=car&weighting=fastest&elevation=true&use_miles=false&layer=OpenStreetMap

https://graphhopper.com/maps/?point=50.549126%2C-4.144668&point=Two%20Bridges%20Hotel%2C%20PL20%206SW%2C%20West%20Devon%2C%20United%20Kingdom&locale=en-US&vehicle=car&weighting=fastest&elevation=true&use_miles=false&layer=OpenStreetMap

I suspect its because of a barrier in the path, please check below link which shows the barrier path:

http://35.246.8.72:8989/maps/?point=50.559019%2C-4.083613&point=50.557272%2C-4.069059&locale=en-US&vehicle=car&weighting=fastest&elevation=false&use_miles=false&layer=OpenStreetMap

**Comment by karussell** [comment]
Submit a new issue only if you are sure it is a missing feature or a bug. For questions or if you are unsure discuss the topic in our forum https://discuss.graphhopper.com
